doc/getting-started/building-from-source.rst

```diff
@@ -181,7 +181,7 @@ You can also build ACRN with your customized scenario:
 
   .. code-block:: none
 
-    $ make BOARD=nuc6cayh SCENARIO=/path/to/scenario.xml
+    $ make BOARD=$PWD/misc/config_tools/data/nuc6cayh/nuc6cayh.xml SCENARIO=/path/to/scenario.xml
```

Comment:
You do not need to give the full path to the board file since you're using the one provided by default.
```suggestion
    $ make BOARD=nuc6cayh SCENARIO=/path/to/scenario.xml
```